backend/src/services/secret-v2-bridge/secret-v2-bridge-fns.ts

```
@@ -411,7 +393,10 @@ export const fnSecretBulkDelete = async ({
     deletedSecrets
```

Are we sure we want to use `Promise.allSettled()` here? If one of the promises inside fails, `Promises.allSettled` **will fail silently**.

Perhaps we should use `Promise.all()` instead, which will throw an error if any of the promises fail